source/shared/core_stmt.cpp

```diff
@@ -2293,27 +2303,39 @@ void finalize_output_parameters( _Inout_ sqlsrv_stmt* stmt TSRMLS_DC )
                 str_len = output_param->original_buffer_len - null_size;
             }
 
-            // if it's not in the 8 bit encodings, then it's in UTF-16
-            if( output_param->encoding != SQLSRV_ENCODING_CHAR && output_param->encoding != SQLSRV_ENCODING_BINARY ) {
-				bool converted = convert_zval_string_from_utf16(output_param->encoding, value_z, str_len);
```

Comment:
You resolved this conversation but forgot to hit reply I think.

Reply:
This is the only place where the old function was called, which was just a wrapper, as you said. Since the converted string may need to be formatted, it is done before copying to the `zval` value.